.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
As part of the project requirements, kindly ensure that your create a file named `test.md`, and add it to the .gitignore here. :wink: